docs/source/api/user-guide/programming-framework/supporting-elements.rst

```diff
@@ -195,14 +195,13 @@ Transformations don't make use of a measure, but measurements do have an ``outpu
 
 .. _max-divergence:
 
-A concrete example is ``MaxDivergence<f64>``,
-read as "the max divergence metric where numbers are expressed in terms of 64-bit floats."
+A concrete example is ``MaxDivergence``, read as "the max divergence privacy measure."
```

Comment:
Confirm that "measure" is better here than "metric"?

Reply:
Right, max divergence is a privacy measure, not a metric.